wgpu/examples/mipmap/main.rs

```diff
@@ -61,15 +61,14 @@ struct Example {
 }
 
 impl Example {
-    fn generate_matrix(aspect_ratio: f32) -> cgmath::Matrix4<f32> {
-        let mx_projection = cgmath::perspective(cgmath::Deg(45f32), aspect_ratio, 1.0, 1000.0);
-        let mx_view = cgmath::Matrix4::look_at_rh(
-            cgmath::Point3::new(0f32, 0.0, 10.0),
-            cgmath::Point3::new(0f32, 50.0, 0.0),
-            cgmath::Vector3::unit_z(),
+    fn generate_matrix(aspect_ratio: f32) -> glam::Mat4 {
+        let projection = glam::Mat4::perspective_rh(consts::FRAC_PI_4, aspect_ratio, 1.0, 1000.0);
```

Comment:
do you remember if the projected depth of this glam function is for 0 to 1, or for -1 to 1?

Reply:
[According to the doc](https://docs.rs/glam/latest/glam/f32/struct.Mat4.html#method.perspective_rh), it's from 0 to 1. There is `perspective_rh_gl` for -1 to 1.